core/shared/src/main/scala-2/me/shadaj/scalapy/py/Module.scala

```diff
@@ -3,6 +3,7 @@ package me.shadaj.scalapy.py
 import scala.language.dynamics
 import scala.reflect.ClassTag
 
+import me.shadaj.scalapy.readwrite.FacadeReader
```

Comment:
Extra import?

Reply:
I have removed this trait since it can be merged into the shared part after defining a polymorphic `create`. I forgot to do it before.